### DISCUSSION TITLE
How do you use this?

### DISCUSSION THREAD

**Discussion by movepixels** [Bacon/BaconQrCode#164] [Q&A]
There are no instructions on how to add a URL and Email?

```
$renderer = new ImageRenderer(
      new RendererStyle(400),
      new SvgImageBackEnd()
    );
    $writer = new Writer($renderer);
    $qr =  $writer->writeString($this->phone);
```

I can write the phone number or the email or the URL but how can you combine all of these?

**Comment by arxeiss** [comment]
What you are asking for is irrelevant to the library. But you need a vCard for that.

I normally use some generator like this: https://www.qr-code-generator.com/solutions/vcard-qr-code/ and then scan the QR code to get the format. And then just do the same in my code.
